app/src/main/java/com/dessalines/thumbkey/utils/Utils.kt

```diff
@@ -1193,12 +1193,15 @@ fun lastColKeysToFirst(board: KeyboardC): KeyboardC {
 
 fun circularDirection(
     positions: List<Offset>,
-    keySize: Double,
+    threshold: Float,
```

Comment:
Same, Int. You can always do `toFloat()` If in the calculation below if necessary.